dso/model.py

```diff
@@ -370,6 +387,17 @@ def key(self) -> str:
         return self.finding.key
 
 
+@dataclasses.dataclass(frozen=True)
+class SastFinding(Finding):
+    sast_statuses: list[SastStatus] | SastStatus
+    component_context: str
```

Comment:
As also mentioned in other [comment](https://github.com/open-component-model/delivery-service/pull/270#discussion_r1881691225), I think we can drop the `component_context` property.